dts/bindings/regulator/regulator-fixed.yaml

```diff
@@ -1,7 +1,8 @@
 # Copyright 2019-2020 Peter Bigot Consulting, LLC
+# Copyright 2023 EPAM Systems
```

Comment:
nit: seems excessive for one line change

Reply:
in the next commit I added more changes